lib/orwell/models/post.ex

```diff
@@ -38,13 +44,43 @@ defmodule Orwell.Post do
     yaml_front_matter(post) <> "\n" <> body <> "\n"
   end
 
+  @spec formatted_utc_today() :: String.t
+  def formatted_utc_today() do
+    Date.utc_today
+    |> Timex.format!("{0D}/{0M}/{YYYY}")
+  end
+
   @spec yaml_front_matter(t) :: String.t
-  def yaml_front_matter(%__MODULE__{title: title}) do
+  def yaml_front_matter(%__MODULE__{
+    id: id,
+    path: path,
+    title: title,
+    author: author,
+    date: date,
+    cover_url: cover_url,
+    retina_cover_url: retina_cover_url,
+    tags: tags,
+    intro: intro
+  }) do
     """
     ---
+    id: #{id}
+    path: #{path}
     title: "#{title}"
+    author: #{author}
+    date: #{date}
+    cover: #{cover_url}
+    retina_cover: #{retina_cover_url}
+    tags:#{tags |> String.split(",") |> yaml_list}
+    intro: "#{intro}"
     ---
     """
   end
-end
 
+  @spec yaml_list(List) :: String.t
+  defp yaml_list(list) do
+    list
+    |> Enum.join("\n  - ")
+    |> String.replace_prefix("", "\n  - ")
```

Comment:
That newline can just be added to the `yaml_front_matter` method itself, but whatever